src/plasma/CMakeLists.txt

```diff
@@ -51,40 +66,53 @@ if(APPLE)
                      POST_BUILD COMMAND ${CMAKE_INSTALL_NAME_TOOL} -change ${PYTHON_SHARED_LIBRARY} ${PYTHON_LIBRARIES} libplasma.so)
 endif(APPLE)
 
-target_link_libraries(plasma ${COMMON_LIB} ${PYTHON_LIBRARIES} ${FLATBUFFERS_STATIC_LIB})
+if(APPLE)
```

Comment:
The statements inside if the 

`if(APPLE)`
`else(APPLE)`
`endif(APPLE)`

should be indented two spaces